tests/rkt_tests.go

```diff
@@ -146,6 +148,12 @@ func (ctx *rktRunCtx) cleanup() {
 		os.Remove("/run/rkt/metadata-svc.sock")
 	}
 
+	for _, child := range ctx.children {
```

Comment:
Is it not assumed that ctx.cleanup() is called before ctx.reset()? does it make sense to run reset() without cleanup()? if not, we should have a variable for tracking the context's state and run cleanup() if only reset() is invoked.